prow/plugins/verify-owners/verify-owners.go

```diff
@@ -483,7 +483,8 @@ func nonTrustedUsersInOwners(ghc githubClient, log *logrus.Entry, triggerConfig
 // and then checks if the owner is a trusted user.
 func checkIfTrustedUser(ghc githubClient, log *logrus.Entry, triggerConfig plugins.Trigger, owner, patch, fileName, org, repo string, nonTrustedUsers map[string][]string, repoAliases repoowners.RepoAliases) (map[string][]string, error) {
 	// only consider owners in the current patch
-	if !strings.Contains(patch, owner) {
+	newOwnerRe, _ := regexp.Compile(fmt.Sprintf(`\+.*%s`, owner))
```

Comment:
Maybe something smarter than just `.*` to avoid matches like `jeanmichel` when you want `michel`?
I realize it wasn't the case before... but maybe a potential improvement.

Reply:
:+1: 

This is also tricker because if a PR contains more than just OWNERS file changes, this regex will include all lines are that are being added :/

Reply:
> This is also tricker because if a PR contains more than just OWNERS file changes, this regex will include all lines are that are being added :/

This PR will actually narrow down the search to valid list entries instead of doing a simple grep on the patch as it was done previously.